recipes/whisper-cpp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(test_package CXX) # if the project uses c++
+
+find_package(whisper-cpp REQUIRED CONFIG)
+find_package(Threads REQUIRED)
+
+set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -pthread")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
```

Comment:
If the library needs `-pthread`, this should be set in the `package_info`'s `self.cpp_info.system_libs`